nautobot/ipam/api/serializers.py

```diff
@@ -401,7 +401,7 @@ class IPAddressSerializer(TaggedObjectSerializer, StatusModelSerializerMixin, Cu
     )
     assigned_object = serializers.SerializerMethodField(read_only=True)
     nat_inside = NestedIPAddressSerializer(required=False, allow_null=True)
-    nat_outside = NestedIPAddressSerializer(read_only=True)
+    nat_outside = NestedIPAddressSerializer(read_only=True, many=True)
```

Comment:
This changes the API so that instead of returning a single object it returns a list of objects - technically that's a breaking change. It might be better from a compatibility perspective if `nat_outside` were to remain as currently implemented (perhaps returning the first nat_outside value in the case where multiples now exist) and add a new `nat_outside_list` key that returns the listing.